src/platform/EFR32/BLEManagerImpl.cpp

```diff
@@ -249,6 +257,8 @@ void BLEManagerImpl::bluetoothStackEventHandler(void * p_arg)
                 RAIL_GetVersion(&railVer, true);
                 ChipLogProgress(DeviceLayer, "RAIL version:, v%d.%d.%d-b%d", railVer.major, railVer.minor, railVer.rev,
                                 railVer.build);
+                sl_bt_connection_set_default_parameters(BLE_CONFIG_MIN_INTERVAL, BLE_CONFIG_MAX_INTERVAL, BLE_CONFIG_LATENCY,
```

Comment:
Is this related to NVM/KVS somehow?

Reply:
Yes, I have seen some BLE supervisor timeouts with the refactor. This allows a longer timeout/interval for ble communication. I did improve the storage API after that so it is probably not needed anymore but this doesn't hurt either way